modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -401,6 +403,20 @@ const FrontendDataSet = ({
 					handleApiError({data, statusCode});
 				}
 				else {
+					setCreationMenu((currentCreationMenu) => {
```

Comment:
@markocikos We could check `currentCreationMenu` presence here or let the `filterCreationAction` handle it.

Reply:
I think it's cleaner if we check here. But I don't have a strong opinion, if you prefer the way it is now, :+1: